app/javascript/register.js

```diff
@@ -0,0 +1,322 @@
+
+// document.addEventListener('turbo:load', () => {
+//     const form = document.querySelector('form');
+//     if (form) {
+//       form.addEventListener('submit', function(event) {
+//         // Clear existing error messages
+//         const errorMessages = document.querySelectorAll('.error-message');
+//         errorMessages.forEach(msg => msg.remove());
+  
+//         let valid = true;
+  
+//         // Get form fields
+//         const firstName = document.getElementById('user_first_name');
+//         const lastName = document.getElementById('user_last_name');
+//         const username = document.getElementById('user_username');
+//         const email = document.getElementById('user_email');
+//         const password = document.getElementById('user_password');
+//         const passwordConfirmation = document.getElementById('user_password_confirmation');
```

Comment:
Don't merge these comments into the main codebase

Reply:
will fixed in next commit